src/compiler/component-lazy/generate-system.ts

```diff
@@ -67,5 +71,6 @@ if (win.__stencil_cssshim) {
 } else {
   start();
 }
+}).call(window);
```

Comment:
do we need call() here?
can't it be `})()`?

Reply:
That's what I've tried initially, but it turned out that the function scope was not bound to `window` when using `})()`. This did break the embedded ES6Promise polyfill unfortunately. Using `.call(window)` therefore explicitly sets the function scope to the `window` object, making the ES6Promise polyfill work again.